### PR TITLE
Doc/hfss sweep examples

### PR DESCRIPTION
**Pull request by MaxJPRey**
Rename the setup for the test `create_frequency_sweep`.
This will avoid confusion with the other tests as every test will have its own _Setup_ under _Analysis_.

Add examples for the following Hfss methods:
`create_linear_count_sweep()`
`create_linear_step_sweep()`

### CODE REVIEW

File: pyaedt/hfss.py [hunk 1]
```diff
@@ -459,8 +460,24 @@ def create_linear_count_sweep(self, setupname, unit, freqstart, freqstop, num_of
...
+            Sweep object if successful. ``False`` if unsuccessful.
+
+        Examples
+        --------
+
+        Create a setup named ``'LinearCountSetup'``.
+        Create a linear count sweep named ``'LinearCountSweep'`` for the analysis setup
+        created above.
+
+        >>> setup = hfss.create_setup("LinearCountSetup")
```
Comment (akaszynski) [inline pyaedt/hfss.py:473]: Good example.

File: pyaedt/hfss.py [hunk 2]
```diff
@@ -459,8 +460,24 @@ def create_linear_count_sweep(self, setupname, unit, freqstart, freqstop, num_of
...
-            Sweep object.
+        :class:`pyaedt.modules.SetupTemplates.SweepHFSS` or bool
+            Sweep object if successful. ``False`` if unsuccessful.
+
+        Examples
+        --------
+
+        Create a setup named ``'LinearCountSetup'``.
+        Create a linear count sweep named ``'LinearCountSweep'`` for the analysis setup
+        created above.
```
Comment (akaszynski) [inline pyaedt/hfss.py:471]:
```suggestion
        Create a setup named ``'LinearCountSetup'`` and use it in a linear count sweep
        named ``'LinearCountSweep'``.
```
I think this reads better.  Please use a similar description in `create_frequency_sweep`

Comment (MaxJPRey) [reply pyaedt/hfss.py:471]: Sure, I will change all the similar comments in my other examples.

**Review by akaszynski — APPROVED** [review]
Perfect.  Good examples.  Let's plan to get an entire module's examples documented and then evaluate where we are at that point.

There's 3000+ methods to document (though not all need it), and pyansys team's goal won't be to do them all on our own.  Regardless, this is a great start and will serve as the foundation of examples moving forward.

**Review by jleonatti — APPROVED** [review]

**Pull request merged**